src/style.css

```diff
@@ -2,0 +1,86 @@
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+  padding: 0 8px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
This line is where you should be specifying your fallback fonts. If 'Avenir' fails to load, the browser will try to use 'Arial', then 'Helvetica', and finally 'sans-serif' if none of the previous fonts are available. The correct format would be `font-family: 'Avenir', 'Avenir-Heavy', Arial, Helvetica, sans-serif;`